dds_registration/models.py

```diff
@@ -192,10 +200,17 @@ def __str__(self):
 
 
 class RegistrationOption(models.Model):
+    SUPPORTED_CURRENCIES = [
+        ("USD", "US Dollar"),
+        ("CHF", "Swiss Franc"),
+        ("EUR", "Euro"),
+        ("CAD", "Canadian Dollar"),  # AKA Loonie :)
+    ]
+
     event = models.ForeignKey(Event, on_delete=models.CASCADE)
     item = models.TextField(null=False, blank=False)  # Show as an input
     price = models.FloatField(default=0, null=False)
-    add_on = models.BooleanField(default=False)
```

Comment:
You have to explain to me how should I deal with options list for the registration: in which way should I change current (a bit complex) code, which filters options by `add_on` property, prepares different data on the backend and shows them as a different ui groups for the user.

Reply:
https://github.com/Depart-de-Sentier/dds_registration/issues/63:

> The table RegistrationOption should only be for registering for an event. A single person can only register once for an event. The column add_on can be removed. 

Simplify the code to force users to choose one and only one registration option (one UI group). The previous approach was complex with adding useful functionality.